RecoTauTag/RecoTau/plugins/PATTauDiscriminationAgainstElectronMVA6.cc

```diff
@@ -0,0 +1,204 @@
+/* class PATTauDiscriminationAgainstElectronMVA6
+ * created : Apr 14 2016,
+ * revised : ,
+ * Authorss :  Anne-Catherine Le Bihan (IPHC)
+ */
+
+#include "FWCore/Utilities/interface/Exception.h"
+#include "FWCore/ParameterSet/interface/FileInPath.h"
+
+#include "RecoTauTag/RecoTau/interface/TauDiscriminationProducerBase.h"
+#include "RecoTauTag/RecoTau/interface/AntiElectronIDMVA6.h"
+
+#include "DataFormats/Math/interface/deltaR.h"
+#include "DataFormats/PatCandidates/interface/Tau.h"
+#include "DataFormats/PatCandidates/interface/PATTauDiscriminator.h"
+#include "DataFormats/Candidate/interface/Candidate.h"
+
+#include <iostream>
+#include <sstream>
+#include <fstream>
+
+using namespace pat;
+
+class PATTauDiscriminationAgainstElectronMVA6 : public PATTauDiscriminationProducerBase  
+{
+ public:
+  explicit PATTauDiscriminationAgainstElectronMVA6(const edm::ParameterSet& cfg)
+    : PATTauDiscriminationProducerBase(cfg),
+      mva_(0),
+      category_output_()
+  {
+    mva_ = new AntiElectronIDMVA6(cfg);
+
+    srcElectrons = cfg.getParameter<edm::InputTag>("srcElectrons");
+    electronToken = consumes<pat::ElectronCollection>(srcElectrons);
+    verbosity_ = ( cfg.exists("verbosity") ) ?
+      cfg.getParameter<int>("verbosity") : 0;
+
+    // add category index
+    produces<PATTauDiscriminator>("category");
+  }
+
+  void beginEvent(const edm::Event&, const edm::EventSetup&);
+
+  double discriminate(const TauRef&) const;
+
+  void endEvent(edm::Event&);
+
+  ~PATTauDiscriminationAgainstElectronMVA6()
+  {
+    delete mva_;
+  }
+
+private:
+  bool isInEcalCrack(double) const;
+
+  std::string moduleLabel_;
+
+  AntiElectronIDMVA6* mva_;
+  float* mvaInput_;
```

Comment:
has been removed. This is left over garbage that had been collected in 81X and also now in 80X.